packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -97,10 +97,18 @@ export const SimpleMenu = ({ label, children, id, as: asComp, ...props }) => {
   const Component = asComp || Button;
 
   useEffect(() => {
-    // Useful to focus the selected item in the list
-    const defaultItemIndexToFocus = childrenArray.findIndex((c) => c.props.children === label);
-    if (defaultItemIndexToFocus !== -1) {
-      setFocusItem(defaultItemIndexToFocus);
+    if (['string', 'number'].includes(typeof label)) {
+      // Useful to focus the selected item in the list
+      const defaultItemIndexToFocus = childrenArray.findIndex((c) => c.props.children === label);
+
+      if (defaultItemIndexToFocus !== -1) {
+        setFocusItem(defaultItemIndexToFocus);
+      }
+    } else if (typeof label === 'object') {
```

Comment:
@HichamELBSI I'm not sure about the check, but what do you think about this? I feel like this works properly in all cases.

Reply:
Maybe you can check if the object as `$$typeof` key?

Reply:
Maybe you can use `React.isValidElement(label)` instead?